src/main/java/com/connexta/store/config/StoreControllerConfiguration.java

```diff
@@ -10,13 +10,23 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.web.multipart.commons.CommonsMultipartResolver;
 
 @Configuration
 public class StoreControllerConfiguration {
 
+  private static final long TEN_GIGABYTES = 10737418240L;
+
   @Bean
   public String storeApiVersion(
       @NotBlank @Value("${endpoints.store.version}") String storeApiVersion) {
     return storeApiVersion;
   }
+
```

Comment:
❓ Shouldn't this already be set by a property in the application.yml? https://github.com/connexta/ion-store/blob/e007a56b48aa6476955357372496ba9061fa6b18/src/main/resources/application.yml#L17

Reply:
Yeah I removed this guy.